theano/tensor/signal/pool.py

```diff
@@ -113,30 +114,103 @@ def pool_2d(input, ds, ignore_border=None, st=None, padding=(0, 0),
     return tensor.reshape(output, outshp, ndim=input.ndim)
 
 
-class Pool(OpenMPOp):
+def pool_3d(input, ds, ignore_border=None, st=None, padding=(0, 0, 0),
```

Comment:
@lamblin @abergeron, what do you think? Should we take this opportunity to change the default of pool_2d now or we need to keep the warning for longer?